app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/mysong/subfragment/favoritesong/FavoriteRecyclerAdapter.java

```diff
@@ -9,6 +9,7 @@
 import android.widget.TextView;
 import com.fstyle.androidtrainning.R;
 import com.fstyle.androidtrainning.data.local.roomdb.entity.TrackEntity;
+import com.fstyle.androidtrainning.data.local.storage.ExternalData;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.data.local.storage.ExternalData. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports